src/Command/ProjectCommands.php

```diff
@@ -39,22 +39,24 @@ public function projectTestCypress(array $args)
             return $result;
         }
         $this->io()->success('Installation of test dependencies successful.');
-
-        $cypress = $this->taskExec('CYPRESS_baseUrl="http://$DKTL_PROXY_DOMAIN" npx cypress run')
+        $config = file_exists("src/tests/cypress.json") ? '--config-file src/tests/cypress.json' : '';
```

Comment:
```suggestion
        $config = file_exists("src/tests/cypress.json") ? ' --config-file src/tests/cypress.json' : '';
```
An extra space is needed at the beginning of this option to keep the command from failing.